test/cypress/e2e/activity.spec.js

```diff
@@ -124,44 +137,53 @@ describe('Dashboard Activity', () => {
     describe(testSet.name, () => {
       beforeEach(() => {
         cy.autologin(accounts.delegate.passphrase, networks.devnet.node);
+        testSet.open();
+        waitBeforeChangeTabAfterLoading();
+        cy.get(ss.delegateStatisticsTab).click();
```

Comment:
The testing talk at React Day was talking about not doing this - each `it` should contain the setup, if it's repetitive like this, create a helper function and call it in `it`.